### PR TITLE
fix: support legacy arm platforms

### PR DESCRIPTION
**Pull request by moelasmar**
*Issue #, if available:*
https://github.com/aws/aws-lambda-builders/issues/348

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws_lambda_builders/workflows/python_pip/packager.py [hunk 1]
```diff
@@ -192,6 +192,9 @@ class DependencyBuilder(object):
         "manylinux1_x86_64": "manylinux_2_5_x86_64",
         "manylinux2010_x86_64": "manylinux_2_12_x86_64",
         "manylinux2014_x86_64": "manylinux_2_17_x86_64",
+        "manylinux1_aarch64": "manylinux_2_5_aarch64",
+        "manylinux2010_aarch64": "manylinux_2_12_aarch64",
+        "manylinux2014_aarch64": "manylinux_2_17_aarch64",
```
Comment (jfuss) [inline aws_lambda_builders/workflows/python_pip/packager.py:197]: Where did you get these mapping from?

**Review by jfuss — CHANGES_REQUESTED** [review]

**Review by mndeveci — CHANGES_REQUESTED** [review]

File: aws_lambda_builders/workflows/python_pip/packager.py [hunk 2]
```diff
@@ -295,6 +298,7 @@ def _download_dependencies(self, directory, requirements_filename):
         incompatible_wheels = set()
         sdists = set()
         for package in deps:
+            print(f">>>>> package.filename {package.filename}")
```
[the remark this answers is not in the document]
Comment (moelasmar) [reply aws_lambda_builders/workflows/python_pip/packager.py:301]: I was doing some testing, and I marked this PR as draft, but seems I changed to ready by mistake .. sorry for that

File: aws_lambda_builders/workflows/python_pip/packager.py (hunk 1 above)
Comment (moelasmar) [reply aws_lambda_builders/workflows/python_pip/packager.py:197]: I was using this PR to test if my assumption is correct or not, and I marked it as draft, but seems I made it ready by mistake .. sorry for that .. I still need to work on it, it is not finalized

File: aws_lambda_builders/workflows/python_pip/packager.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mndeveci) [reply aws_lambda_builders/workflows/python_pip/packager.py:301]: Please don't be, just let us know once it is ready to be reviewed again :)

**Pull request closed without merging**